docs/advanced-features/using-mdx.md

```diff
@@ -0,0 +1,166 @@
+---
+description: Learn how to use @next/mdx in your Next.js project.
+---
+
+# Using MDX with Next.js
+
+Next.js supports MDX through a number of different means, this page will outline some of the ways you can begin integrating MDX into your Next.js project.
+
+## What is MDX?
+
+MDX is a superset of markdown that lets you write JSX directly in your markdown files. It is a powerful way to add dynamic interactivity, and embed components within your content, helping you to bring your pages to life.
+
+### MDX plugins
+
+Internally MDX uses remark and rehype. Remark is a markdown processor powered by a plugins ecosystem. This plugin ecosystem lets you parse code, transform `HTML` elements, change syntax, extract frontmatter, and more.
+
+Rehype is an `HTML` processor, also powered by a plugin ecosystem. Similar to remark, these plugins let you manipulate, sanitize, compile and configure all types of data, elements and content.
+
+To use a plugin from either remark or rehype, you will need to add it to the MDX packages config.
+
+## `@next/mdx`
+
+The `@next/mdx` package is configured in the `next.config.js` file at your projects root. **It sources data from local files**, allowing you to create pages with a `.mdx` extension, directly in your `/pages` directory.
+
+### Setup `@next/mdx` in Next.js
+
+The following steps outline how to setup `@next/mdx` in your Next.js project:
+
+1. Install the required packages:
+
+   ```terminal
+     npm install @next/mdx @mdx-js/loader
+
+     #or
```

Comment:
Let's just teach NPM first 👍

Reply:
Do we not give both options in other paces?

Reply:
We've been moving towards NPM first since most folks know the syntax if they're going for yarn, and specifically on GitHub, there's now a copy code button in snippets, so having just one choice is better.

Not a blocking comment, just a passing thought 👍

Reply:
TIL